is_core/generic_views/table_views.py

```diff
@@ -126,6 +126,9 @@ def _get_export_display(self):
     def _get_list_display_extra(self):
         return list(self.list_display_extra)
 
+    def _get_list_per_page(self):
+        return self.list_per_page or IS_CORE_LIST_PER_PAGE
+
```

Comment:
Na TableViewMixinu se bere nejdřív to, co je nastaveno na view a pak to co je v settings. Tohle se vlastně používá v inlinech.